transforms/language/text_encoder/python/README.md

```diff
@@ -0,0 +1,47 @@
+# Text Encoder Transform 
+
+## Summary 
+This transform is using sentence encoder models to create embeddings of the text in each row of the input .parquet table.
+
+## Running
+
+### Parameters
+
+The transform can be tuned with the following parameters.
+
+
+| Parameter  | Default  | Description  |
+|------------|----------|--------------|
+| `model_name`                    | `BAAI/bge-small-en-v1.5` | The HF model to use for encoding the text. |
```

Comment:
If this is currently restricted to hf models, maybe call this key `hf_model_name`?  This might allow more flexibility going forward to add new model types/providers.  I like the generic `model_name`, but what happens when we use other model types.   Another option is to require the value to include the indicator.  For example `hf:BAAI/bge-small-en-v1.5`

Reply:
if we use a different provider, very likely we need to have a different execution code, because the `model_name` is used by the HF `transforms` library directly.

If/when we get to other providers, I think we will need a "executor factory" and a corresponding parameter to change it.